src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/OtlpLogRecordProcessor.cs

```diff
@@ -16,22 +16,33 @@
 
 using System.Runtime.CompilerServices;
 using Google.Protobuf;
+using OpenTelemetry.Internal;
 using OpenTelemetry.Logs;
+using OpenTelemetry.Trace;
 using OtlpCollector = OpenTelemetry.Proto.Collector.Logs.V1;
 using OtlpCommon = OpenTelemetry.Proto.Common.V1;
 using OtlpLogs = OpenTelemetry.Proto.Logs.V1;
 using OtlpResource = OpenTelemetry.Proto.Resource.V1;
 
 namespace OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation;
 
-internal static class LogRecordExtensions
+internal sealed class OtlpLogRecordProcessor
```

Comment:
nit: The term "Processor" is overloaded with OTel's notion of a processor. I suggest something like `OtlpLogRecordTransformer`.